js/client.js

```diff
@@ -15,13 +14,8 @@ window.EmojiKeyboard = {
     preapreEmojiKeyboard: function() {
         const that = this;
         $.each(emojis, function (category) {
-            let elementID = category.toLowerCase(),
-                anyEmojiSupported = 0;
-            const emojiCategoryDOM = $('<p>', {
-                class: 'emoji-category',
-                id: elementID,
-            });
 
+            $emojies = "";
```

Comment:
Why `$` sign? Also missing `let` declaration before variable and it should be `emojis` not `emojies`.